src/main/java/favourite/use_case/UserReaderInterface.java

```diff
@@ -0,0 +1,15 @@
+package favourite.use_case;
+
+import entities.User;
+
+import java.util.ArrayList;
+
+public interface UserReaderInterface {
```

Comment:
These are actually needed though I believe, to allow the data access to 'create' users